test/unit/controllers/CommodityCodeBestMatchControllerSpec.scala

```diff
@@ -33,8 +33,9 @@ class CommodityCodeBestMatchControllerSpec extends ControllerSpecBase {
 
   val formProvider = new CommodityCodeBestMatchFormProvider()
   val form: Form[Boolean] = formProvider()
+  val goodsName = "goods"
```

Comment:
We could also change this string to something else and do what @tipperba did here: https://github.com/hmrc/binding-tariff-trader-frontend/pull/204/files#diff-01a5235a65d0e40c32e0876ca64f3d89R57

By using "goods" we aren't actually verifying that the goods name is being read from the cache since that is equal to our default.